content/docs/jsx-in-depth.md

```diff
@@ -265,30 +265,30 @@ const App = () => {
 };
 ```
 
-In the example above, the `kind` prop is safely consumed and *is not* passed on to the `<button>` element in the DOM.
-All other props are passed via the `...other` object making this component really flexible. You can see that it passes an `onClick` and `children` props.
+Pada contoh di atas, *jenis* (`kind`) *prop* tersebut secara aman digunakan dan *tidak* dioper kepada elemen `<button>` dalam DOM.
```

Comment:
```suggestion
Pada contoh di atas, *prop* `kind` tersebut secara aman digunakan dan *tidak* dioper kepada elemen `<button>` dalam DOM.
```

This refers to the `kind` prop, not *the kind of a prop*. 😅

Reply:
>Pada contoh di atas, *prop* `kind` tersebut secara aman digunakan dan *tidak* dioper kepada elemen `<button>` dalam DOM.

This one is my lack of knowledge 😅 Thank you